testutils/sims/runner.go

```diff
@@ -0,0 +1,229 @@
+package sims
+
+import (
+	"fmt"
+	"io"
+	"path/filepath"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/client"
+
+	"cosmossdk.io/log"
```

Comment:
```suggestion
	"cosmossdk.io/core/log"
```

Reply:
The runner sets up the logger via `log.NewTestLogger(t)` for example. This is not available in core/log